### DISCUSSION TITLE
Sort Stuff!

### DISCUSSION THREAD

**Discussion by Norb3** [modrinth/.github#258] [Feature requests]
It seems that none of the lists on the web site (in particular collections, search results, and the follow list) are not, and cannot be, sorted. This lack makes usage more difficult so the capability should be added.

**Comment by Renzaccio** [comment]
I agree, adding a sorting system similar to the search results would be very useful.
It could be added to the collections (including the _Followed_ collection) and also to the users' pages.

**Comment by Renzaccio** [comment]
@Norb3 I would suggest you to rename your post to something like _Adding Sorting System to Collection, Followed List and User's Pages_.
It would make it easier for the devs to indentify your request.
